### PR TITLE
2 about page

### LINKED ISSUE #2: About Page

### PR DESCRIPTION
**Pull request by iman-om**
# Related Issue

- Resolve #2

### CODE REVIEW

File: src/components/aboutus/AboutUs.jsx [hunk 1]
```diff
@@ -128,81 +224,32 @@ const AboutUs = () => {
...
-                                                    size={20}
-                                                    className='text-gray-700'
-                                                />
-                                            </a>
-                                        </div>
-                                    </div>
-                                </div>
-                            ))}
-                        </div>
+                        <TeamComponent team={team} />
```
Comment (HachemBouhadede) [inline src/components/aboutus/AboutUs.jsx:227]: not sure why it's not working !

**Review by HachemBouhadede — COMMENTED** [review]
image

image

tech logo and khalil's avatar aren't being displayed coorectly

File: src/components/aboutus/AboutUs.jsx (hunk 1 above)
Comment (iman-om) [reply src/components/aboutus/AboutUs.jsx:227]: i will make sure to fix it, thank you for ur remarks

File: public/locales/ar/common.json [hunk 2]
```diff
@@ -33,7 +33,7 @@
 
     "HeroSection": {
         "title": "انضموا إلينا لإحداث تغيير مؤثر",
-        "introduction": "مرحبًا بكم في 'ايد ب ايد'، حيث نجمع بين القلوب والأيدي لإحداث فارق. تُعَدّ فعاليات الخير لدينا أكثر من مجرد تجمعات؛ إنها فرص لتحويل الحياة. انضموا إلى مهمتنا لإلهام الأمل، وتعزيز التغيير، ونشر العطاء. معًا، يمكننا صنع مستقبل أفضل، حدثا بعد حدث."
+        "introduction": "مرحبًا بكم في 'ايد ب ايد'، حيث نجمع بين القلوب والأيدي لإحداث فارق. تُعَدّ فعاليات الخير لدينا أكثر من مجرد تجمعات؛ إنها فرص لتحويل الحياة. انضموا إلى مهمتنا لإلهام الأمل، وتعزيز التغيير، ونشر العطاء. معًا، يمكننا صنع مستقبل أفضل، حدثا بعد حدث.",
```
Comment (Khalil-NOUI) [inline public/locales/ar/common.json:36]: conflict resolve

**Review by Khalil-NOUI — COMMENTED** [review]
nice, just consider the conflict

**Pull request merged**